docs/quickstart.md

```diff
@@ -0,0 +1,293 @@
+
+This tutorial will introduce the main `nemos` functionalities. This is intended for users that are 
+already familiar with the GLM framework but want to learn how to interact with the `nemos` API. 
+If you have used [scikit-learn](https://scikit-learn.org/stable/) before, we are compatible the [estimator API](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html), so the quickstart should look 
+familiar.
+
+In the following sessions, you will learn:
+
+1. How to define and fit a GLM model.
+2. Which are the GLM parameters and hyperparameters and how to set it.
+3. How `nemos` interacts with `pynapple` for pre-processing
+4. How `nemos` interacts with `scikit-learn` for pipelines and cross-validation. 
+
+Each of this section will be an independent example.
+
+### Basic Model Fit
+
+Defining and fitting a `nemos` GLM model is straightforward and can be done using the following syntax:
+
+```python
+import nemos as nmo
+import numpy as np
+
+# predictors, shape (n_samples, n_neurons, n_features)
+X = 0.2 * np.random.normal(size=(100, 1, 1))
+# true coefficients, shape (n_neurons, n_features)
+coef = np.random.normal(size=(1, 1))
+# observed counts, shape (n_samples, n_neurons)
+y = np.random.poisson(np.exp(np.einsum("nf, tnf -> tn", coef, X)))
```

Comment:
I don't think we want to show this in our quickstart -- it's not how most users will interact with nemos. instead, maybe show X and y being loaded from a file with `np.load`, describing their shape in the comment above.

Then, after this code block, say what this assumes about X and y: they have the same number of time points and neurons, y is the output of some spike-sorter, and X includes some experimenter-described features, like animal location or stimulus. Then say "see below for a brief example using pynapple to do some common standard manipulations" or the like.

Reply:
I think load requires more time to explain the variables, which is not the point of this. instead simulation to me is a nice way to not focus on the data but on how the code works. also Guillaume argues that it is good to see how one could simulate data